### ISSUE TITLE
NWVars NW2Vars

### ISSUE THREAD

**Issue by d4kir92** [Facepunch/garrysmod-issues#5140]
### Details
any branch, latest version
i made an admin mod, but in the scoreboard some nw2vars are not showing the right values
is it because of "pvs"?
what should i use for showing for example usergroup string NWVars? NW2Vars? or something else?

### Steps to reproduce
server:
ply:SetNW2String( "ug", "superadmin" )

client
ply:GetNW2String( "ug", "NOTSET" )

for some players the usergroup shows "NOTSET"

**Comment by robotboy655** [comment]
Do not use NW2Vars. They were made available for testing, and should only be used for such.

**Comment by d4kir92** [comment]
ohhhh, thanks for the info! will use nwvars then.
i started using them because NWVars wasnt working for some reason a time ago, atleast for me.

**Comment by Kefta** [comment]
Preferably you should use the net library since you are exclusively sending data outside of prediction. The stock usergroup nwvar is a convenience garry-ism.

**Comment by thegrb93** [comment]
This happened so many times when I used to host.

**Comment by d4kir92** [comment]
yeah, switch some things to net library and only using nwvars helped! thanks guys 😄
